regulations/static/regulations/css/less/module/comment.less

```diff
@@ -94,20 +94,29 @@ Preamble Read Mode
   .section-title,
   .subpart-addition,
   .subpart-info {
-    width: 75%;
+    width: 100%;
   }
```

Comment:
I think we can remove this rule (and 75% rule for `cfr-instructions`) altogether; div's expand to full width without explicitly being told to. Now that everything is contained within the 75% div, we should be good to go.